scripts/lcov.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
```

Comment:
Minor: not a big fan of the commit structure here. In the future, please start with an identical copy, and make changes as a separate commit. This makes it much easier to see what changed.

As a general rule, moves (of files, functions, or large chunks of code), as well a large whitespace changes (re-indent / fix code style), should be separated as much as possible from actual changes. This makes it much easier to review what's going on.

(No need to rewrite history here at this point, but please keep in mind for the future.)